tailwind.config.js

```diff
@@ -24,19 +24,24 @@ module.exports = {
     },
     extend: {
       colors: {
-        border: 'hsl(var(--border))',
-        input: 'hsl(var(--input))',
-        ring: 'hsl(var(--ring))',
-        background: 'hsl(var(--background))',
-        foreground: 'hsl(var(--foreground))',
+        brand: {
+          100: '#33332E',
+          600: '#09131D',
```

Comment:
Using `hsl` format seems to have some advantages for OLED screens. But, if you feel more comfortable with HEX is also fine.